src/components/Typography/__tests__/Typography.spec.js

```diff
@@ -0,0 +1,30 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/vue';
+import { H1, H2, H3, P, Caption } from '../';
+
+const c = (name, value) => ({ name, value });
+
+const components = [
+  c('H1', H1),
+  c('H2', H2),
+  c('H3', H3),
+  c('P', P),
+  c('Caption', Caption)
+];
+
+for (const component of components) {
+  describe('typography', () => {
+
+    it('renders correctly', () => {
+      let content = 'This is text that will render inside the component';
+      if (component.value === H3) {
```

Comment:
This isn't really necessary. Since your CSS is controlling the visual presentation (not changing the actual Unicode characters) of your h3s, this test will pass perfectly without manually making the content uppercase.

Reply:
Huh, good to know.